### PR TITLE
Export date range inclusive

### PR DESCRIPTION
**Pull request by neoliteconsultant**
Closes #

#### Does this change require updates to documentation? If so, please file an issue at https://github.com/opendatakit/docs/issues/new and include the link below.

### CODE REVIEW

File: src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java [hunk 1]
```diff
@@ -56,438 +56,438 @@
 
 @SuppressWarnings("checkstyle:MethodName")
 public class ConfigurationPanelForm extends JComponent {
-  public JPanel container;
```
Comment (ggalmazor) [inline src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java]: You have change indentation on this file from 2 spaces to 4 spaces. Could you revert that, please? If you're using IntelliJ, there is a code style definition file in `config/codestyle/codestyle.xml`. You can import that to ensure this doesn't happen.

File: src/org/opendatakit/briefcase/operations/Export.java [hunk 2]
```diff
@@ -15,24 +15,6 @@
  */
 package org.opendatakit.briefcase.operations;
 
-import static java.time.format.DateTimeFormatter.ISO_DATE_TIME;
```
Comment (ggalmazor) [inline src/org/opendatakit/briefcase/operations/Export.java]: It seems like this changeset is only about imports ordering. Please, drop this commit.

File: src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java (hunk 1 above)
Comment (neoliteconsultant) [reply src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java]: Sure. I will do that

File: src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.form [hunk 3]
```diff
@@ -31,7 +31,7 @@
           <gridbag weightx="0.0" weighty="0.0"/>
         </constraints>
         <properties>
-          <text value="Start date (inclusive)"/>
+          <text value="  Start date (inclusive)"/>
```
Comment (ggalmazor) [inline src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.form]: Why these spaces? If it's to "hack" text alignment, maybe you can try to align the text to the right instead? You may need to assign the text some fixed witdh for that to work.

Comment (neoliteconsultant) [reply src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.form]: Noted. I will remove the spaces

**Comment by neoliteconsultant** [comment]
I have squashed my commits

**Comment by lognaturel** [comment]
Thanks, @neoliteconsultant! It looks like the squash didn't quite work as expected because I'm seeing a bunch of commits here. You can read more about how to squash at https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

It's not a big deal in this case because a committer can squash on merge. That's typically what we do for 1-commit pull requests anyway so that there's a linear history and no merge commits (this is a slightly controversial practice). So if in doubt, maybe you can practice doing interactive rebases on a practice branch and not worry about this one. The diff is exactly what's desired which is what really matters. 👍

**Comment by neoliteconsultant** [comment]
Thanks  @lognaturel.

**Comment by ggalmazor** [comment]
Hi @neoliteconsultant!

It seems that your branch has "branched off" from our master branch and now it has a lot of different commits...

I've downloaded your fork and I think it would be enough to create a new fresh branch from opendatakit's repo's latest master branch, cherrypick your commit c9a2b0c, push it to your fork and create a new PR.

To do the cherry pick you can execute this command: `git cherry-pick c9a2b0c05c9dc5767d58c53cf8faed127a24c40a`.

**Comment by neoliteconsultant** [comment]
@ggalmazor sure I will do that

**Comment by neoliteconsultant** [comment]
@ggalmazor I have created a branch 'inclusive_label_export_date' and initiated a new pull request as suggested

**Comment by kkrawczyk123** [comment]
I think that it doesn't work in End Date context. It looks good with Start Date but if I want to export some submissions from today I have to choose tomorrow as End Date - master has the same behavior - I thought this would change in this pr so correct me if I am wrong. 
The same behavior on all three systems: Ubuntu, MacOS and Windows.

**Comment by ggalmazor** [comment]
@kkrawczyk123, I've checked the source code and the range is inclusive on both ends. If you get less submissions than expected maybe there is a bug :P

**Comment by kkrawczyk123** [comment]
What is expected then? I had 3 submissions from today.

**Comment by ggalmazor** [comment]
if you set the end date to today, you should get those 3 submissions exported.

**Comment by kkrawczyk123** [comment]
This is exactly what I was expecting but I set yesterday's date as start date and today's as an end date, but only yesterday's submissions are exported. Today's are exported when I set tomorrow as an end date.

**Comment by ggalmazor** [comment]
Alright! Then we should open a new issue for this :)

**Comment by kkrawczyk123** [comment]
Testing it in #477 

@opendatakit-bot unlabel "needs testing"

**Comment by lognaturel** [comment]
Closing in favor of #477.

**Pull request closed without merging**